imager/imager_images/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.conf import settings
+
+# Create your models here.
+
+
+@python_2_unicode_compatible
+class Album(models.Model):
+    name = models.CharField(max_length=30)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                            on_delete=models.CASCADE,
+                            related_name='albums',
+                            null=True)
+
```

Comment:
"Meta-data should include a title and a description. Also a date created, date modified and date published as well as a published field containing the same options described for Photos. Users should be able to designate one contained photo as the ‘cover’ for the album."